docs/source/conf.py

```diff
@@ -147,9 +147,16 @@
 
 # TODO: remove this once https://github.com/pytorch/pytorch_sphinx_theme/issues/125 is fixed
 html_css_files = [
+    "https://cdn.datatables.net/1.10.23/css/jquery.dataTables.min.css",
     "css/custom_torchvision.css",
 ]
 
+html_js_files = [
+    "https://cdn.datatables.net/1.10.23/js/jquery.dataTables.min.js",
```

Comment:
Is there a preferred CDN for pytorch? Like CDN js? Or something which Meta uses and relies on.

Reply:
Perhaps this is something @svekars would know?

Reply:
An alternative is to download this file and keep in our repo. This will avoid CDN.